### ISSUE TITLE
Allow 'read more' expanded context

### ISSUE THREAD

**Issue by Cogiva** [biybl/biybl#11]
Allow users to click on verses to read more - whole chapter etc.

**Comment by gerv** [comment]
Yep. I imagine a "read more..." link at the bottom, which switches to a view which is simply a single chapter, perhaps with the original passage highlighted, and then "next" and "previous" buttons. This is not going to be a full-blown Bible browser with "select book", jump-based navigation and so on, but it should allow moving forwards or backwards from the current passage.

The use case here is someone stuck in a service they don't understand, who is enraptured by the word of God in their language and wants to keep going!

**Comment by Cogiva** [comment]
Highlight verses that have been expanded.

**Comment by Cogiva** [comment]
No flow across books in expanded view
